python/cudf/cudf/__init__.py

```diff
@@ -84,6 +86,8 @@
     pivot,
     unstack,
 )
+
+# TODO: Pandas does not expose isclose in the root namespace.
```

Comment:
This function was added to address a specific need. With pandas you can just use `numpy.isclose` in all cases. However, using `cupy.isclose` doesn't work when a `cudf.Series` contains nulls, so this wrapper is necessary. You can see the discussion on this in #5105 and #5442.

Reply:
Thanks. I'm removing the TODO and adding `isclose` to the public API in `__all__`.